velox/docs/functions/spark/aggregate.rst

```diff
@@ -0,0 +1,13 @@
+===================
+Aggregate Functions
+===================
+
+Aggregate functions operate on a set of values to compute a single result.
+
+General Aggregate Functions
+---------------------------
+
+.. function:: last(x [,ignoreNull]) -> x
```

Comment:
SQL functions do not support optional arguments. Hence, this need to be documented as two separate functions: last(x) and last(x, ignoreNull).

Reply:
Thanks， just fixed it.